src/Asma/Finra.java

```diff
@@ -0,0 +1,28 @@
+package Asma;
+
+public class Finra {
+    public static void main(String[] args) {
+        Finra(5);
+        Finra(11);
+        Finra(20);
+        Finra(2);
+    }
+    //Write a method which prints out the numbers from 1 to 30 but for numbers which are a multiple of 3,
+    //print "FIN" instead of the number and
+    //for numbers which are a multiple of 5, print "RA" instead of the number.
+    //for numbers which are a multiple of both 3 and 5, print "FINRA" instead of the number.
+    public static void Finra(int num){
```

Comment:
In here you accept int parameter for your method, but you never used it inside the method body. I'm gonna merge it but later on you should change your method signature and create new PR.